examples/jsm/misc/GroundProjectedEnv.js

```diff
@@ -0,0 +1,154 @@
+import * as THREE from "three";
```

Comment:
In the example modules, we normally only import what's required. For this particular use case, it should be `Mesh`,  `IcosahedronGeometry`, `ShaderMaterial` and `DoubleSide`.